tools/accuracy_checker/accuracy_checker/postprocessor/super_resolution_image_recovery.py

```diff
@@ -0,0 +1,72 @@
+"""
+Copyright (c) 2018-2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import cv2
+import numpy as np
+from PIL import Image
+
+from ..config import StringField, NumberField
+from .postprocessor import Postprocessor
+from ..representation import SuperResolutionPrediction, SuperResolutionAnnotation
+from ..utils import get_size_from_config
+
+
+class SRImageRecovery(Postprocessor):
+    __provider__ = 'sr_image_recovery'
+
+    annotation_types = (SuperResolutionAnnotation, )
+    prediction_types = (SuperResolutionPrediction, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'target_color': StringField(optional=True, choices=['bgr', 'rgb'], default='rgb'),
+            'dst_width': NumberField(
+                value_type=int, optional=True, min_value=1, description="Width of model input before recovering"
+            ),
+            'dst_height': NumberField(
+                value_type=int, optional=True, min_value=1, description="Height of model input before recovering"
+            ),
+            'size': NumberField(
+                value_type=int, optional=True, min_value=1,
+                description="Size of model input for both dimensions (height and width) before recovering"
+            )
+        })
+        return parameters
+
+    def configure(self):
+        self.color = cv2.COLOR_YCrCb2BGR if self.get_value_from_config('target_color') == 'bgr' else cv2.COLOR_YCrCb2RGB
+        self.input_height, self.input_width = get_size_from_config(self.config, allow_none=True)
```

Comment:
you allow none but the code below will be crashed in case if self.input_width=None or self.input_height=None

Reply:
fixed